projects/dsp-ui/src/lib/viewer/values/time-value/time-input/time-input.component.spec.ts

```diff
@@ -132,4 +161,62 @@ describe('TimeInputComponent', () => {
         expect(dateTime.time).toEqual('12:00');
 
     });
+
+    it('should mark the form\'s validity correctly', () => {
+        expect(testHostComponent.timeInputComponent.valueRequiredValidator).toBe(true);
+        expect(testHostComponent.timeInputComponent.form.valid).toBe(true);
+
+        testHostComponent.timeInputComponent.timeFormControl.setValue(null);
+
+        testHostComponent.timeInputComponent._handleInput();
+
+        expect(testHostComponent.timeInputComponent.form.valid).toBe(false);
+
+        testHostComponent.timeInputComponent.timeFormControl.setValue("");
+
+        testHostComponent.timeInputComponent._handleInput();
+
+        expect(testHostComponent.timeInputComponent.form.valid).toBe(false);
+    });
+});
+
+describe('TimeInputComponent no value required', () => {
+    let testHostComponent: NoValueRequiredTestHostComponent;
+    let testHostFixture: ComponentFixture<NoValueRequiredTestHostComponent>;
+
+    beforeEach(async(() => {
+        TestBed.configureTestingModule({
```

Comment:
Do you need this setup again?

Reply:
This is how it's done in the other complex value spec files. It could possibly be refactored at a later point but I don't think it's necessary.